analytics_data_api/v0/views/learners.py

```diff
@@ -0,0 +1,308 @@
+"""
+API methods for module level data.
+"""
+from rest_framework import generics, status
+
+from analytics_data_api.constants import (
+    learner
+)
+from analytics_data_api.v0.exceptions import (
+    LearnerEngagementTimelineNotFoundError,
+    LearnerNotFoundError,
+    ParameterValueError,
+)
+from analytics_data_api.v0.models import (
+    ModuleEngagement,
+    ModuleEngagementMetricRanges,
+    RosterEntry
+)
+from analytics_data_api.v0.serializers import (
+    CourseLearnerMetadataSerializer,
+    ElasticsearchDSLSearchSerializer,
+    EngagementDaySerializer,
+    LearnerSerializer,
+)
+from analytics_data_api.v0.views import CourseViewMixin
+from analytics_data_api.v0.views.utils import split_query_argument
+
+
+class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
+    """
+    Get data for a particular learner in a particular course.
+
+    **Example Request**
+
+        GET /api/v0/learners/{username}/?course_id={course_id}
+
+    **Response Values**
+
+        Returns metadata and engagement data for the learner.
+
+            * username: User's username.
+            * enrollment_mode: Enrollment mode (for example, "verified").
+            * name: User's full name.
+            * email: User's email address.
+            * segments: Classification, based on engagement, of this learner's
+              work in this course (for example, "highly_engaged" or
+              "struggling").
+            * engagements: Summary of engagement events for a time span.
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the different problems the learner tried.
```

Comment:
Replace "different" with "individual"? Maybe add a third sentence to clarify that if a learner tries the same problem multiple times, this total will count only one problem attempt?